pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
     'deepecho>=0.6.0',
     'rdt>=1.12.0',
     'sdmetrics>=0.14.0',
+    'platformdirs>=4.0'
```

Comment:
should we add this to the dependency checker since you're not capping it?

Reply:
Yeah, let me add it!